data/spells/scripts/support/food.lua

```diff
@@ -1,19 +1,19 @@
-local food = {
+local foods = {
 	2666, -- meat
 	2671, -- ham
 	2681, -- grape
 	2674, -- apple
 	2689, -- bread
 	2690, -- roll
-	2696  -- cheese
+	2696 -- cheese
 }
 
 function onCastSpell(creature, variant)
-	creature:addItem(food[math.random(#food)])
-	if math.random(1, 100) > 50 then
-		creature:addItem(food[math.random(#food)])
+	if math.random(1, 100) >= 50 then
```

Comment:
You changed the chance from 50% to 51% by replacing `>` with `>=`. Also, as the chance is 50%, you can use `math.random(1, 2) == 1` instead.

Why did you change the order, exactly?